Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+
+RUN mkdir -p /app
+
+COPY . main.py /app/
+
+WORKDIR /app
+
+COPY ./requirements.txt /app/requirements.txt
+
+RUN pip install -r /app/requirements.txt
+
```

Comment:
you can expose our default port number 8000 here, or add ["--port", "8000"] as last params to CMD.

Reply:
I have done as you recommended